scripts/panels.txt

```diff
@@ -0,0 +1,21 @@
+{'altitude': 150.0, 'coordinates': {'latitude': 51.524774, 'longitude': -0.131913}, 'relativeLocation': 'UCL, Institute of Archaeology, South Entrance, London, UK'}
```

Comment:
What's this for?

Reply:
Locations of all the panels. for reference